PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * ONNXRuntime.h
+ *
+ * A convenience wrapper of the ONNXRuntime C++ API.
+ * Based on https://github.com/microsoft/onnxruntime/blob/master/csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/CXX_Api_Sample.cpp.
+ *
+ *  Created on: Jun 28, 2019
+ *      Author: hqu
+ */
+
+#ifndef PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+#define PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+
+#include <vector>
+#include <map>
+#include <string>
+#include <memory>
+
+#include "onnxruntime/core/session/onnxruntime_cxx_api.h"
+
+namespace cms {
+
+  namespace Ort {
+
+    using namespace ::Ort;
```

Comment:
the code rule checker is not happy about this.
This case is inside the local namespace scope and should not be a problem.
However, it's better to make the rule checker happy as it's likely not a big effort here.
Please resolve.

Reply:
Fixed in https://github.com/cms-sw/cmssw/commit/7f10c2a3d3ed06a1c9321001e739d179d8a0043b.